addons/escoria-core/game/core-scripts/esc/commands/spawn.gd

```diff
@@ -1,4 +1,4 @@
-# `spawn path [object2]`
+# `spawn identifier path [object2]`
```

Comment:
Maybe We should add a "is_active" parameter here so you could spawn a scene as an object but hide it first.